bodo/pandas/_plan.cpp

```diff
@@ -109,6 +109,13 @@ duckdb::unique_ptr<duckdb::Expression> make_col_ref_expr(
         ctype, source_cols[col_idx]);
 }
 
+duckdb::unique_ptr<duckdb::Expression> make_function_expr(
+    std::string function_name) {
+    return duckdb::make_uniq<duckdb::BoundColumnRefExpression>(
+        "count_star()", duckdb::LogicalType::BIGINT,
+        duckdb::ColumnBinding(-1, 0), 0);
```

Comment:
Add a check for function_name to be `count_star()` just in case?

Reply:
done